.github/actions/test-dotnet/Invoke-SolutionTests.ps1

```diff
@@ -22,7 +22,7 @@ else
 {
     $connectionStringStem = 'Server=.;Database=LombiqUITestingToolbox_{{id}};User Id=sa;Password=Password1!'
 
-    $Env:Lombiq_Tests_UI__DockerConfiguration__ContainerName = 'sql2019'
+    $Env:Lombiq_Tests_UI__DockerConfiguration__ContainerName = 'sql2022'
```

Comment:
You can set the container name here. This is the answer to your question (https://github.com/Lombiq/GitHub-Actions/pull/293#discussion_r1418781392), value `uitt-sqlserver`.